changelog.d/403.bugfix

```diff
@@ -0,0 +1 @@
+Fix issue with obey_x_forwarded_for not being False when "False".
```

Comment:
```suggestion
Fix issue with `obey_x_forwarded_for` not being honoured.
```

As I think the issue is more that there is no way of setting it to false currently?